tron.bat

```diff
@@ -61,32 +61,48 @@ call :set_cur_date
 :: VARIABLES :: ---------------- These are the defaults. Change them if you want ------------------- ::
 :::::::::::::::
 :: Rules for variables:
-::  * NO quotes!                    (bad:  "c:\directory\path"       )
-::  * NO trailing slashes on paths! (bad:   c:\directory\            )
-::  * Spaces are okay               (okay:  c:\my folder\with spaces )
-::  * Network paths are okay        (okay:  \\server\share name      )
+::  * No trailing slashes on paths!   (BAD:  SET VAR=C:\Directory\)
+::  * BAD quotes!                     (BAD:  SET VAR="C:\Directory")
+::  * Network paths are okay          (okay: SET VAR=\\SERVER\Share)
+::  * Spaces are okay                 (okay: SET VAR=C:\My Folder\With Spaces)
+::  * BEST quotes with spaces is best (BEST: SET "VAR=C:\My Folder\With Spaces")
+::  *                                 (BEST: SET "VAR=\\SERVER\Share With Spaces")
 
 :: By DEFAULT, LOGPATH is the parent directory for all of Tron's output (logs, backups, etc). Tweak the paths below to your liking if you want to change it
 :: If you want a separate directory generated per Tron run (for example when doing multiple runs for testing), use something like this:
 ::   set LOGPATH=%SystemDrive%\Logs\tron\%computername%_%DTS%
-set LOGPATH=%SystemDrive%\Logs\tron
+set "LOGPATH=%SystemDrive%\Logs\tron"
 
 :: Master log file. To differentiate logfiles if you're doing multiple runs, you can set this to something like:
 ::  set LOGFILE=tron_%COMPUTERNAME%_%DTS%.log
 set LOGFILE=tron.log
 
 :: Where Tron should save files that the various virus scanners put in quarantine. Currently unused (created, but nothing is stored here)
-set QUARANTINE=%LOGPATH%\quarantine
+set "QUARANTINE=%LOGPATH%\quarantine"
 
 :: Registry, Event Logs, and power scheme backups are all saved here
-set BACKUPS=%LOGPATH%\backups
+set "BACKUPS=%LOGPATH%\backups"
 
 :: Where to save raw unprocessed logs from the various sub-tools
-set RAW_LOGS=%LOGPATH%\raw_logs
+set "RAW_LOGS=%LOGPATH%\raw_logs"
 
 :: Where to save the summary logs
-set SUMMARY_LOGS=%LOGPATH%\summary_logs
+set "SUMMARY_LOGS=%LOGPATH%\summary_logs"
 
+		:: Where resume state and flags are stored
```

Comment:
Forgot I indented these lines before I was fully setup with how github works, won't be an issue.